lib/Doctrine/ruleset.xml

```diff
@@ -375,6 +375,7 @@
         <exclude name="Squiz.Arrays.ArrayDeclaration.ValueNotAligned"/>
         <!-- Checked by SlevomatCodingStandard.Arrays.TrailingArrayComma.MissingTrailingComma -->
         <exclude name="Squiz.Arrays.ArrayDeclaration.NoCommaAfterLast"/>
+        <exclude name="Squiz.Arrays.ArrayDeclaration.NoComma"/>
```

Comment:
Maybe a comment explaining the reason?

Reply:
The comment is the one above :smile: (L376)